x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/fields/field_types/geo_shape_type.tsx

```diff
@@ -65,15 +65,16 @@ export const GeoShapeType = ({ field }: Props) => {
           description={i18n.translate(
             'xpack.idxMgmt.mappingsEditor.geoShape.pointsOnlyFieldDescription',
             {
-              defaultMessage: 'Configures the geo_shape field type for point shapes only.',
+              defaultMessage:
+                'This optimizes index and search performance when it is known that only points will be indexed. Shapes, including multi-point shapes, will be rejected.',
```

Comment:
Re "Shapes, including multi-point shapes, will be rejected" -- I don't see this in the docs. Am I missing it? My suggestion below leaves this sentence out but if I'm wrong then I think it makes sense to include it.

Here's my suggestion for making this a bit terser:

> Optimizes geo-shape queries as long as this field only contains geo-points.

Reply:
Optimizes geo-shape queries if this field contains only geo-points.

Let's be consistent.  Are we using "Optimizes" or "Optimize" to start the sentences.